src/main/java/com/chatapp/vartalaab/configuration/CustomWebSocketHandler.java

```diff
@@ -0,0 +1,81 @@
+package com.chatapp.vartalaab.configuration;
+
+import com.chatapp.vartalaab.dto.MessageDto;
+import com.chatapp.vartalaab.service.MessageService;
+import com.chatapp.vartalaab.service.UserSessionService;
+import com.chatapp.vartalaab.utils.GeneralUtility;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.socket.CloseStatus;
+import org.springframework.web.socket.TextMessage;
+import org.springframework.web.socket.WebSocketSession;
+import org.springframework.web.socket.handler.TextWebSocketHandler;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Slf4j
+public class CustomWebSocketHandler extends TextWebSocketHandler {
```

Comment:
This class is using spring components, but this class itself is not spring component ?